tests/Strategies/CallbackMaskingStrategyTest.php

```
@@ -234,19 +235,19 @@ public function testCallbackReceivesOriginalValue(): void
 
     public function testCallbackCanReturnNonString(): void
     {
-        $callback = fn(mixed $value): array => ['masked' => true];
+        $callback = fn(mixed $value): array => [TestConstants::DATA_MASKED => true];
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$value'. (undefined)

(UnusedFormalParameter)